### ISSUE TITLE
jvm metrics not respect scraping interval

### ISSUE THREAD

**Issue by sophieyfang** [GoogleCloudPlatform/ops-agent#301]
**Description**
JVM metrics is sending out at a frequency every 5 seconds resulting in an error denied by Cloud Monitoring API side:
```
2021-12-17T01:44:55.104Z	info	exporterhelper/queued_retry.go:215	Exporting failed. Will retry the request after interval.	{"kind": "exporter", "name": "googlecloud", "error": "rpc error: code = InvalidArgument desc = One or more TimeSeries could not be written: One or more points were written more frequently than the maximum sampling period configured for the metric.: timeSeries[0-7]\nerror details: name = Unknown  desc = total_point_count:8  errors:{status:{code:9}  point_count:8}", "interval": "5.52330144s"}
```

** To Reproduce**

```
sophieyfang_google_com@debian10-meow:~$ cat otel.yaml 
exporters:
  googlecloud:
    metric:
      prefix: ""
    user_agent: Google-Cloud-Ops-Agent-Metrics/latest (BuildDistro=build_distro;Platform=linux;ShortName=linux_platform;ShortVersion=linux_platform_version)
  logging:
    loglevel: debug
processors:
  filter/jvm:
    metrics:
      include:
        match_type: strict
        metric_names:
        - jvm.memory.pool.used
  resourcedetection/_global_0:
    detectors:
    - gce
  metricstransform/jvm__pipeline_jvm__metrics_1:
    transforms:
    - action: update
      include: ^(.*)$$
      match_type: regexp
      new_name: workload.googleapis.com/$${1}
  normalizesums/jvm__pipeline_jvm__metrics_0: {}
receivers:
  jmx/jvm__pipeline_jvm__metrics:
    collection_interval: 30s
    endpoint: service:jmx:rmi:///jndi/rmi://127.0.0.1:9010/jmxrmi
    jar_path: /opt/google-cloud-ops-agent/subagents/opentelemetry-collector/opentelemetry-java-contrib-jmx-metrics.jar
    target_system: jvm
service:
  pipelines:
    metrics/jvm__pipeline_jvm__metrics:
      exporters:
      - googlecloud
      - logging
      processors:
      - filter/jvm
      receivers:
      - jmx/jvm__pipeline_jvm__metrics
(reverse-i-search)`': cat otel.yaml ^C
sophieyfang_google_com@debian10-meow:~$ 
sophieyfang_google_com@debian10-meow:~$ /opt/google-cloud-ops-agent/subagents/opentelemetry-collector/otelopscol --add-instance-id=false --config=otel.yaml
2021-12-17T01:43:47.870Z	info	service/collector.go:174	Applying configuration...
2021-12-17T01:43:47.878Z	info	builder/exporters_builder.go:259	Exporter was built.	{"kind": "exporter", "name": "googlecloud"}
2021-12-17T01:43:47.878Z	info	builder/exporters_builder.go:259	Exporter was built.	{"kind": "exporter", "name": "logging"}
2021-12-17T01:43:47.879Z	info	filterprocessor@v0.37.1-0.20211028205244-e6fab4102b84/filter_processor.go:73	Metric filter configured	{"kind": "processor", "name": "filter/jvm", "include match_type": "strict", "include expressions": [], "include metric names": ["jvm.memory.pool.used"], "include metrics with resource attributes": null, "exclude match_type": "", "exclude expressions": [], "exclude metric names": [], "exclude metrics with resource attributes": null}
2021-12-17T01:43:47.879Z	info	builder/pipelines_builder.go:220	Pipeline was built.	{"pipeline_name": "metrics/jvm__pipeline_jvm__metrics", "pipeline_datatype": "metrics"}
2021-12-17T01:43:47.879Z	info	builder/receivers_builder.go:228	Receiver was built.	{"kind": "receiver", "name": "jmx/jvm__pipeline_jvm__metrics", "datatype": "metrics"}
2021-12-17T01:43:47.880Z	info	service/service.go:86	Starting extensions...
2021-12-17T01:43:47.880Z	info	service/service.go:91	Starting exporters...
2021-12-17T01:43:47.880Z	info	builder/exporters_builder.go:40	Exporter is starting...	{"kind": "exporter", "name": "googlecloud"}
2021-12-17T01:43:47.882Z	info	builder/exporters_builder.go:48	Exporter started.	{"kind": "exporter", "name": "googlecloud"}
2021-12-17T01:43:47.882Z	info	builder/exporters_builder.go:40	Exporter is starting...	{"kind": "exporter", "name": "logging"}
2021-12-17T01:43:47.882Z	info	builder/exporters_builder.go:48	Exporter started.	{"kind": "exporter", "name": "logging"}
2021-12-17T01:43:47.882Z	info	service/service.go:96	Starting processors...
2021-12-17T01:43:47.882Z	info	builder/pipelines_builder.go:52	Pipeline is starting...	{"pipeline_name": "metrics/jvm__pipeline_jvm__metrics", "pipeline_datatype": "metrics"}
2021-12-17T01:43:47.883Z	info	builder/pipelines_builder.go:63	Pipeline is started.	{"pipeline_name": "metrics/jvm__pipeline_jvm__metrics", "pipeline_datatype": "metrics"}
2021-12-17T01:43:47.883Z	info	service/service.go:101	Starting receivers...
2021-12-17T01:43:47.883Z	info	builder/receivers_builder.go:68	Receiver is starting...	{"kind": "receiver", "name": "jmx/jvm__pipeline_jvm__metrics"}
2021-12-17T01:43:47.884Z	info	otlpreceiver/otlp.go:68	Starting GRPC server on endpoint 0.0.0.0:40973	{"kind": "receiver", "name": "jmx/jvm__pipeline_jvm__metrics"}
2021-12-17T01:43:47.885Z	info	builder/receivers_builder.go:73	Receiver started.	{"kind": "receiver", "name": "jmx/jvm__pipeline_jvm__metrics"}
2021-12-17T01:43:47.885Z	info	service/telemetry.go:92	Setting up own telemetry...
2021-12-17T01:43:47.886Z	info	service/telemetry.go:116	Serving Prometheus metrics	{"address": ":8888", "level": "basic", "service.instance.id": "", "service.version": "latest"}
2021-12-17T01:43:47.886Z	info	service/collector.go:230	Starting google-cloud-metrics-agent...	{"Version": "latest", "NumCPU": 1}
2021-12-17T01:43:47.886Z	info	service/collector.go:132	Everything is ready. Begin running and processing data.
2021-12-17T01:43:52.897Z	INFO	loggingexporter/logging_exporter.go:56	MetricsExporter	{"#metrics": 1}
2021-12-17T01:43:52.900Z	DEBUG	loggingexporter/logging_exporter.go:66	ResourceMetrics #0
Resource labels:
     -> service.name: STRING(unknown_service:java)
     -> telemetry.sdk.language: STRING(java)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.version: STRING(1.7.0)
InstrumentationLibraryMetrics #0
InstrumentationLibrary io.opentelemetry.contrib.jmxmetrics 1.6.0
Metric #0
Descriptor:
     -> Name: jvm.memory.pool.used
     -> Description: current memory pool usage
     -> Unit: by
     -> DataType: Gauge
NumberDataPoints #0
Data point attributes:
     -> name: STRING(Eden Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:43:52.319 +0000 UTC
Value: 16259944
NumberDataPoints #1
Data point attributes:
     -> name: STRING(CodeHeap 'profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:43:52.319 +0000 UTC
Value: 4356992
NumberDataPoints #2
Data point attributes:
     -> name: STRING(Metaspace)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:43:52.319 +0000 UTC
Value: 7129432
NumberDataPoints #3
Data point attributes:
     -> name: STRING(Survivor Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:43:52.319 +0000 UTC
Value: 20392
NumberDataPoints #4
Data point attributes:
     -> name: STRING(Tenured Gen)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:43:52.319 +0000 UTC
Value: 2503816
NumberDataPoints #5
Data point attributes:
     -> name: STRING(CodeHeap 'non-nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:43:52.319 +0000 UTC
Value: 1212928
NumberDataPoints #6
Data point attributes:
     -> name: STRING(CodeHeap 'non-profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:43:52.319 +0000 UTC
Value: 1880320
NumberDataPoints #7
Data point attributes:
     -> name: STRING(Compressed Class Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:43:52.319 +0000 UTC
Value: 770600

2021-12-17T01:44:18.838Z	INFO	loggingexporter/logging_exporter.go:56	MetricsExporter	{"#metrics": 1}
2021-12-17T01:44:18.838Z	DEBUG	loggingexporter/logging_exporter.go:66	ResourceMetrics #0
Resource labels:
     -> service.name: STRING(unknown_service:java)
     -> telemetry.sdk.language: STRING(java)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.version: STRING(1.7.0)
InstrumentationLibraryMetrics #0
InstrumentationLibrary io.opentelemetry.contrib.jmxmetrics 1.6.0
Metric #0
Descriptor:
     -> Name: jvm.memory.pool.used
     -> Description: current memory pool usage
     -> Unit: by
     -> DataType: Gauge
NumberDataPoints #0
Data point attributes:
     -> name: STRING(Eden Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:18.829 +0000 UTC
Value: 16259944
NumberDataPoints #1
Data point attributes:
     -> name: STRING(CodeHeap 'profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:18.829 +0000 UTC
Value: 4356992
NumberDataPoints #2
Data point attributes:
     -> name: STRING(Metaspace)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:18.829 +0000 UTC
Value: 7129432
NumberDataPoints #3
Data point attributes:
     -> name: STRING(Survivor Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:18.829 +0000 UTC
Value: 20392
NumberDataPoints #4
Data point attributes:
     -> name: STRING(Tenured Gen)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:18.829 +0000 UTC
Value: 2503816
NumberDataPoints #5
Data point attributes:
     -> name: STRING(CodeHeap 'non-nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:18.829 +0000 UTC
Value: 1212928
NumberDataPoints #6
Data point attributes:
     -> name: STRING(CodeHeap 'non-profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:18.829 +0000 UTC
Value: 1880320
NumberDataPoints #7
Data point attributes:
     -> name: STRING(Compressed Class Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:18.829 +0000 UTC
Value: 770600

2021-12-17T01:44:23.043Z	INFO	loggingexporter/logging_exporter.go:56	MetricsExporter	{"#metrics": 1}
2021-12-17T01:44:23.043Z	DEBUG	loggingexporter/logging_exporter.go:66	ResourceMetrics #0
Resource labels:
     -> service.name: STRING(unknown_service:java)
     -> telemetry.sdk.language: STRING(java)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.version: STRING(1.7.0)
InstrumentationLibraryMetrics #0
InstrumentationLibrary io.opentelemetry.contrib.jmxmetrics 1.6.0
Metric #0
Descriptor:
     -> Name: jvm.memory.pool.used
     -> Description: current memory pool usage
     -> Unit: by
     -> DataType: Gauge
NumberDataPoints #0
Data point attributes:
     -> name: STRING(Eden Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:23.033 +0000 UTC
Value: 497824
NumberDataPoints #1
Data point attributes:
     -> name: STRING(CodeHeap 'profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:23.033 +0000 UTC
Value: 4361856
NumberDataPoints #2
Data point attributes:
     -> name: STRING(Metaspace)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:23.033 +0000 UTC
Value: 7129432
NumberDataPoints #3
Data point attributes:
     -> name: STRING(Survivor Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:23.033 +0000 UTC
Value: 47816
NumberDataPoints #4
Data point attributes:
     -> name: STRING(Tenured Gen)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:23.033 +0000 UTC
Value: 2503816
NumberDataPoints #5
Data point attributes:
     -> name: STRING(CodeHeap 'non-nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:23.033 +0000 UTC
Value: 1212928
NumberDataPoints #6
Data point attributes:
     -> name: STRING(CodeHeap 'non-profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:23.033 +0000 UTC
Value: 1880320
NumberDataPoints #7
Data point attributes:
     -> name: STRING(Compressed Class Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:23.033 +0000 UTC
Value: 770600

2021-12-17T01:44:48.835Z	INFO	loggingexporter/logging_exporter.go:56	MetricsExporter	{"#metrics": 1}
2021-12-17T01:44:48.835Z	DEBUG	loggingexporter/logging_exporter.go:66	ResourceMetrics #0
Resource labels:
     -> service.name: STRING(unknown_service:java)
     -> telemetry.sdk.language: STRING(java)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.version: STRING(1.7.0)
InstrumentationLibraryMetrics #0
InstrumentationLibrary io.opentelemetry.contrib.jmxmetrics 1.6.0
Metric #0
Descriptor:
     -> Name: jvm.memory.pool.used
     -> Description: current memory pool usage
     -> Unit: by
     -> DataType: Gauge
NumberDataPoints #0
Data point attributes:
     -> name: STRING(Eden Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:48.828 +0000 UTC
Value: 497824
NumberDataPoints #1
Data point attributes:
     -> name: STRING(CodeHeap 'profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:48.828 +0000 UTC
Value: 4361856
NumberDataPoints #2
Data point attributes:
     -> name: STRING(Metaspace)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:48.828 +0000 UTC
Value: 7129432
NumberDataPoints #3
Data point attributes:
     -> name: STRING(Survivor Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:48.828 +0000 UTC
Value: 47816
NumberDataPoints #4
Data point attributes:
     -> name: STRING(Tenured Gen)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:48.828 +0000 UTC
Value: 2503816
NumberDataPoints #5
Data point attributes:
     -> name: STRING(CodeHeap 'non-nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:48.828 +0000 UTC
Value: 1212928
NumberDataPoints #6
Data point attributes:
     -> name: STRING(CodeHeap 'non-profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:48.828 +0000 UTC
Value: 1880320
NumberDataPoints #7
Data point attributes:
     -> name: STRING(Compressed Class Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:48.828 +0000 UTC
Value: 770600

2021-12-17T01:44:53.144Z	INFO	loggingexporter/logging_exporter.go:56	MetricsExporter	{"#metrics": 1}
2021-12-17T01:44:53.144Z	DEBUG	loggingexporter/logging_exporter.go:66	ResourceMetrics #0
Resource labels:
     -> service.name: STRING(unknown_service:java)
     -> telemetry.sdk.language: STRING(java)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.version: STRING(1.7.0)
InstrumentationLibraryMetrics #0
InstrumentationLibrary io.opentelemetry.contrib.jmxmetrics 1.6.0
Metric #0
Descriptor:
     -> Name: jvm.memory.pool.used
     -> Description: current memory pool usage
     -> Unit: by
     -> DataType: Gauge
NumberDataPoints #0
Data point attributes:
     -> name: STRING(Eden Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:53.136 +0000 UTC
Value: 954968
NumberDataPoints #1
Data point attributes:
     -> name: STRING(CodeHeap 'profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:53.136 +0000 UTC
Value: 4361856
NumberDataPoints #2
Data point attributes:
     -> name: STRING(Metaspace)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:53.136 +0000 UTC
Value: 7129432
NumberDataPoints #3
Data point attributes:
     -> name: STRING(Survivor Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:53.136 +0000 UTC
Value: 47816
NumberDataPoints #4
Data point attributes:
     -> name: STRING(Tenured Gen)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:53.136 +0000 UTC
Value: 2503816
NumberDataPoints #5
Data point attributes:
     -> name: STRING(CodeHeap 'non-nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:53.136 +0000 UTC
Value: 1212928
NumberDataPoints #6
Data point attributes:
     -> name: STRING(CodeHeap 'non-profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:53.136 +0000 UTC
Value: 1880320
NumberDataPoints #7
Data point attributes:
     -> name: STRING(Compressed Class Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:53.136 +0000 UTC
Value: 770600

^C2021-12-17T01:44:55.042Z	info	service/collector.go:146	Received signal from OS	{"signal": "interrupt"}
2021-12-17T01:44:55.042Z	info	service/collector.go:242	Starting shutdown...
2021-12-17T01:44:55.042Z	info	service/service.go:121	Stopping receivers...
2021-12-17T01:44:55.069Z	INFO	loggingexporter/logging_exporter.go:56	MetricsExporter	{"#metrics": 1}
2021-12-17T01:44:55.070Z	DEBUG	loggingexporter/logging_exporter.go:66	ResourceMetrics #0
Resource labels:
     -> service.name: STRING(unknown_service:java)
     -> telemetry.sdk.language: STRING(java)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.version: STRING(1.7.0)
InstrumentationLibraryMetrics #0
InstrumentationLibrary io.opentelemetry.contrib.jmxmetrics 1.6.0
Metric #0
Descriptor:
     -> Name: jvm.memory.pool.used
     -> Description: current memory pool usage
     -> Unit: by
     -> DataType: Gauge
NumberDataPoints #0
Data point attributes:
     -> name: STRING(Eden Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:55.053 +0000 UTC
Value: 954968
NumberDataPoints #1
Data point attributes:
     -> name: STRING(CodeHeap 'profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:55.053 +0000 UTC
Value: 4361856
NumberDataPoints #2
Data point attributes:
     -> name: STRING(Metaspace)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:55.053 +0000 UTC
Value: 7129432
NumberDataPoints #3
Data point attributes:
     -> name: STRING(Survivor Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:55.053 +0000 UTC
Value: 47816
NumberDataPoints #4
Data point attributes:
     -> name: STRING(Tenured Gen)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:55.053 +0000 UTC
Value: 2503816
NumberDataPoints #5
Data point attributes:
     -> name: STRING(CodeHeap 'non-nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:55.053 +0000 UTC
Value: 1212928
NumberDataPoints #6
Data point attributes:
     -> name: STRING(CodeHeap 'non-profiled nmethods')
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:55.053 +0000 UTC
Value: 1880320
NumberDataPoints #7
Data point attributes:
     -> name: STRING(Compressed Class Space)
StartTimestamp: 1970-01-01 00:00:00 +0000 UTC
Timestamp: 2021-12-17 01:44:55.053 +0000 UTC
Value: 770600

2021-12-17T01:44:55.089Z	error	subprocess/subprocess.go:249	subprocess died	{"kind": "receiver", "name": "jmx/jvm__pipeline_jvm__metrics", "error": "unexpected shutdown: exit status 130"}
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver/internal/subprocess.(*Subprocess).run
	/root/go/pkg/mod/github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver@v0.37.1-0.20211028205244-e6fab4102b84/internal/subprocess/subprocess.go:249
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver/internal/subprocess.(*Subprocess).Start.func1
	/root/go/pkg/mod/github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver@v0.37.1-0.20211028205244-e6fab4102b84/internal/subprocess/subprocess.go:127
2021-12-17T01:44:55.089Z	info	service/service.go:126	Stopping processors...
2021-12-17T01:44:55.089Z	info	builder/pipelines_builder.go:71	Pipeline is shutting down...	{"pipeline_name": "metrics/jvm__pipeline_jvm__metrics", "pipeline_datatype": "metrics"}
2021-12-17T01:44:55.089Z	info	builder/pipelines_builder.go:75	Pipeline is shutdown.	{"pipeline_name": "metrics/jvm__pipeline_jvm__metrics", "pipeline_datatype": "metrics"}
2021-12-17T01:44:55.089Z	info	service/service.go:131	Stopping exporters...
2021-12-17T01:44:55.104Z	info	exporterhelper/queued_retry.go:215	Exporting failed. Will retry the request after interval.	{"kind": "exporter", "name": "googlecloud", "error": "rpc error: code = InvalidArgument desc = One or more TimeSeries could not be written: One or more points were written more frequently than the maximum sampling period configured for the metric.: timeSeries[0-7]\nerror details: name = Unknown  desc = total_point_count:8  errors:{status:{code:9}  point_count:8}", "interval": "5.52330144s"}
2021-12-17T01:44:55.105Z	info	service/service.go:136	Stopping extensions...
2021-12-17T01:44:55.106Z	info	service/collector.go:258


```

**Comment by sophieyfang** [comment]
I've tried using v1.9.0 https://github.com/open-telemetry/opentelemetry-java-contrib/tree/v1.9.0/jmx-metrics. Still no luck. It's populting JVM metrics every 3seconds..

**Comment by sophieyfang** [comment]
Also reported in java-contrib: https://github.com/open-telemetry/opentelemetry-java-contrib/issues/153

**Comment by cpheps** [comment]
@sophieyfang I'll look into the opentelemetry-java-contrib issue. Can you assign me to this please?

**Comment by cpheps** [comment]
I think I found the issue in the JMX java program. Commented on the ticket in the java-contrib repo. It has to do with flushing metrics to collector as well as a timed export. Awaiting response from someone who knows that codebase better to explain why its setup that way in case I'm missing some details on the implementation.

**Comment by cpheps** [comment]
@sophieyfang This should be solved once open-telemetry/opentelemetry-java-contrib#190 is merged

**Comment by sophieyfang** [comment]
Verified that it's fixed in the ops-agent `2.9.0`. Thanks again for putting in a fix on this issue @cpheps 🎉
